pkg/registry/authorization/localsubjectaccessreview/rest.go

```diff
@@ -75,5 +75,11 @@ func (r *REST) Create(ctx context.Context, obj runtime.Object, createValidation
 		localSubjectAccessReview.Status.EvaluationError = evaluationErr.Error()
 	}
 
+	if createValidation != nil {
```

Comment:
move to line 65, after validation, before action

Reply:
done